src/mmap.rs

```diff
@@ -949,4 +1074,32 @@ mod tests {
         assert!(region.file_offset().is_some());
         assert_eq!(region.file_offset().unwrap().start(), offset);
     }
+
+    #[test]
```

Comment:
I am not sure this test adds much, it would fail even for a grammar fix.

Reply:
Yes, that's true.

I can remove it, but then I would have to decrease the coverage with 0.1, because of these 1 added line that I cannot test:  `UnsortedMemoryRegions,`

Reply:
I think that's reasonable.

Reply:
Coverage is just an hint, we needs useful code and tests:)

Reply:
Ok. I removed the tests and decreased the coverage.